internal/charm/repository/package_test.go

```diff
@@ -14,3 +14,7 @@ import (
 func TestPackage(t *testing.T) {
 	gc.TestingT(t)
 }
+
+func ptr[T any](x T) *T {
```

Comment:
I'm starting to think this should be in the stdlibrary.

Reply:
~50 definitions. Probably time to put it in a utility.